docs/beast_grid_inputs.rst

```diff
@@ -119,8 +119,11 @@ Choices:
 
 * `beast.physicsmodel.dust.extinction_extension` (recommended of sub 912 A extinction needed)
    * all the observation based models stop at or before 912 A as hydrogen dominates extinction below this wavelength
-   * for work that interested in sub 912 A information (e.g., ionizing photon measurements), these models extend the
+   * For work that interested in sub 912 A information (e.g., ionizing photon measurements), these models extend the
      dust extinction to shorter than 912 A wavelengths by smoothly merging dust grain models with an observed extinction model
```

Comment:
One last thing -- could you specify the minimum wavelength for the extension here?

Reply:
Done.